### PR TITLE
ENH 11 rc1

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
cc @isuruf

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {% set name = "openmp" %}
-{% set version = "10.0.1" %}
-{% set sha256 = "d19f728c8e04fb1e94566c8d76aef50ec926cd2f95ef3bf1e0a5de4909b28b44" %}
+{% set version = "11.0.0.rc1" %}
+{% set sha256 = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" %}
```
Comment (beckermr) [inline recipe/meta.yaml:3]:
```suggestion
{% set sha256 = "be015908f29747c24633106b384ea55632b269be34bec1a6ed74e014789f9a94" %}
```

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -14,3 +14,6 @@ c_compiler_version:     # [osx]
   - "*"                 # [osx]
 cxx_compiler_version:   # [osx]
   - "*"                 # [osx]
+
```
Comment (isuruf) [inline recipe/conda_build_config.yaml:17]:
```suggestion
channel_sources:
  - conda-forge/label/llvm_rc,conda-forge,defaults
```

Comment (beckermr) [reply recipe/conda_build_config.yaml]: i'll get the rest of my PRs

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by isuruf** [comment]
Pushed to `rc` branch

**Pull request merged**